tree/treeplayer/src/TArrowDS.cxx

```diff
@@ -0,0 +1,314 @@
+// Author: Giulio Eulisse CERN  2/2018
+
+/*************************************************************************
+ * Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.               *
+ * Copyright (C) 2018, Giulio Eulisse.                                   *
```

Comment:
FWIW, that might be problematic.

Reply:
Indeed.  Giulio, could you update to have the same header as the other files?  Thanks.

Reply:
What’s the rule here? I see plenty of different copyright statements in the project and under international law adding code authors copyright statements is the correct thing to do. This of course does not mean that the code is not licensed under ROOT terms.